eng/pipelines/templates/jobs/cmake-generate.tests.yml

```diff
@@ -28,9 +28,13 @@ parameters:
   - name: JobName
     type: string
     default: CMakeGenerate
+  - name: TimeoutInMinutes
+    type: number
+    default: 120
```

Comment:
Looks like the default value of `timeoutInMinutes` is 60 minutes, do we need to bump this up to 2 hours?

Is the reason for this because the test discovery for each test/CMakeLists.txt is sequential, and since we are bumping the test discovery timeout to 10 minutes (from 5 seconds), it is increasing our CI pipeline times?

https://learn.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#timeouts

Reply:
i had it fail a few times in a row due to 1 hr timeout. so i figured since we bump one timeout we might need to bump this one too , the regular builds are already at 2 hours, i bumped them a while back